server/src/QRServer/discord/bot.py

```diff
@@ -312,17 +429,16 @@ async def _basic_validations_passed(
 
         return (True, "")
 
-    async def _send_user_notification(self, message: str) -> None:
-        # Send a message to the user notifications channel
-        if not self.user_notifications_channel_id:
+    async def _send_notification(self, message: str, channel: str) -> None:
```

Comment:
I thought that duplicating this function for other notifications channel would be meh, so now there is a common one.

Reply:
I could but I don't think it's a good idea, especially since the only difference is the target channel